src/schema/v2/artwork/index.ts

```diff
@@ -213,6 +213,27 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           }
         },
       },
+      editionSize: {
+        type: GraphQLString,
+        resolve: ({ edition_sets }) => {
+          if (edition_sets && edition_sets.length === 1) {
+            return edition_sets[0]!.edition_size
+          }
+        },
+      },
+      editionNumber: {
+        type: GraphQLInt,
+        resolve: ({ edition_sets }) => {
+          if (edition_sets && edition_sets.length === 1) {
+            const editionNumber =
+              edition_sets[0]!.available_editions?.[0] ?? null
+
+            if (editionNumber) {
+              return Number(editionNumber)
+            }
+          }
+        },
```

Comment:
Same here, can lean heavily on the `?`: 
```
resolve: ({ edition_sets }) => edition_sets?.[0]?.available_editions?.[0]
```

Reply:
Good catch, not using the nullable coalescing operator...however I don't want to call `Number()` on null so I think a check here for a non-null value is needed first?

Reply:
That makes sense 👍 could maybe still leverage this existential form and assign to a variable that you can do an `if (var)` on to wrap as a Number.